<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>653</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="codeEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>471</width>
      <height>571</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Terminal</family>
      <pointsize>9</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="runButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>88</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Run</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>88</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stepButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>88</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Step</string>
    </property>
   </widget>
   <widget class="QLabel" name="registerALabel">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>60</y>
      <width>20</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>A:</string>
    </property>
   </widget>
   <widget class="QLabel" name="registerBLabel">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>80</y>
      <width>20</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>B:</string>
    </property>
   </widget>
   <widget class="QLabel" name="registerCLabel">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>100</y>
      <width>20</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>C:</string>
    </property>
   </widget>
   <widget class="QLabel" name="registerELabel">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>140</y>
      <width>20</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>E:</string>
    </property>
   </widget>
   <widget class="QLabel" name="registerFLabel">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>160</y>
      <width>20</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>F:</string>
    </property>
   </widget>
   <widget class="QLabel" name="registerDLabel">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>120</y>
      <width>20</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>D:</string>
    </property>
   </widget>
   <widget class="QLabel" name="registerXLabel">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>210</y>
      <width>20</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>X:</string>
    </property>
   </widget>
   <widget class="QLabel" name="registerYLabel">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>230</y>
      <width>20</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Y:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="registerAValue">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>60</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="registerBValue">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>80</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="registerCValue">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>100</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="registerDValue">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>120</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="registerEValue">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>140</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="registerFValue">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>160</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="registerXValue">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>210</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="registerYValue">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>230</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QSlider" name="speedSlider">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>20</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="singleStep">
     <number>10</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="speedLabel">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>0</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Speed</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>260</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Instruction:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="instructionBox">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>280</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="codeStatus">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>310</y>
      <width>47</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="currentSpeed">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>20</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>100</string>
    </property>
   </widget>
   <widget class="QLabel" name="speedSuffix">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>20</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>ms</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="incBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>340</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>inc</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="decBox">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>360</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>dec</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="iszBox">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>380</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>isz</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="jmpBox">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>400</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>jmp</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="stpBox">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>420</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>stp</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="instructionCounter">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>440</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="resetButton">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>10</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>653</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As...</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
